core/modules/startup/render.js

```diff
@@ -29,7 +29,11 @@ var THROTTLE_REFRESH_TIMEOUT = 400;
 
 exports.startup = function() {
 	// Set up the title
-	$tw.titleWidgetNode = $tw.wiki.makeTranscludeWidget(PAGE_TITLE_TITLE,{document: $tw.fakeDocument, parseAsInline: true});
+	$tw.titleWidgetNode = $tw.wiki.makeTranscludeWidget(PAGE_TITLE_TITLE, {
+		document: $tw.fakeDocument,
+		parseAsInline: true,
+		importPageMacros: true,
```

Comment:
hmmm -- Do we really need the whole PageMacro content? I think there should be a more specific tag, to avoid the memory overhead, just to populate browser tabs once.

Reply:
I know, just use `, importVariables: "[[$:/core/macros/lingo]]"`, it accept a filter.

Reply:
Thanks, I think that makes sense.

Reply:
Actually I think it makes more sense to include all the page macros. Otherwise it's confusing to users why some macros are available and not others.

Reply:
@Jermolene Will number of imported macro/variables impact performance significantly?